### DISCUSSION TITLE
After upgrading from 9.0.0-rc.85 to 9.0.0-rc.87, the program reports an error

### DISCUSSION THREAD

**Discussion by zhuyifang** [directus/directus#7116] [Archive]
When you can log in successfully and redirect to: http://127.0.0.1:4000/admin/collections/user_extend/+, the program reports an error:

```json
{
  "errors": [
    {
      "message": "select `directus_fields`.`collection`, `directus_fields`.`display`, `directus_fields`.`display_options`, `directus_fields`.`field`, `directus_fields`.`group`, `directus_fields`.`hidden`, `directus_fields`.`id`, `directus_fields`.`interface`, `directus_fields`.`note`, `directus_fields`.`options`, `directus_fields`.`readonly`, `directus_fields`.`sort`, `directus_fields`.`special`, `directus_fields`.`translations`, `directus_fields`.`width`, `directus_fields`.`conditions` from `directus_fields` order by `directus_fields`.`id` asc - ER_BAD_FIELD_ERROR: Unknown column 'directus_fields.conditions' in 'field list'",
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}
```

**Comment by zhuyifang** [comment]
It seems that the new version has modified the database structure, the field `directus_fields`.`conditions` in the above error does not exist in my previous version of rc.85
Is there a version upgrade guide?

**Reply by keesvanbemmel** [reply]
I think basically you should run database migrations. Which is always a good idea when upgradig, especially while directus is still in RC!

[Upgrade Database](https://docs.directus.io/reference/command-line-interface/#upgrade-the-database)

**Reply by zhuyifang** [reply]
thank you very much
Because English is too bad, so many documents are not read carefully, this is my own problem
